### PR TITLE
Styling/landingpage

### PR DESCRIPTION
**Pull request by tombjarne**
Updated home.js with style changes, navigation, footer and roadmap section.

Minor updates to App.css

Landingpage Navigation added to home.js only

### CODE REVIEW

File: src/components/pages/home.js [hunk 1]
```diff
@@ -9,9 +11,14 @@ import handHeart from "../../assets/img/hand-heart.svg";
 import lockHeart from "../../assets/img/lock-heart.svg";
 
 class Home extends React.Component {
+
   render() {
+
     return (
       <section id="home">
+        <LandingNav />
+        <div id="hero">
```
Comment (santiagoarriaga) [inline src/components/pages/home.js:20]: @tombjarne this currently shows as an empty orange bar, is that right ?

**Comment by tombjarne** [comment]
Sure, what is it you are wondering?

**Comment by tombjarne** [comment]
> @tombjarne looks good to me, just a quick question, thanks !

Sure! What is it you are wondering?

File: src/components/pages/home.js (hunk 1 above)
Comment (tombjarne) [reply src/components/pages/home.js:20]: Yes, it is a placeholder for the content to come.

**Pull request merged**